mlir/test/Conversion/VectorToArmSME/vector-to-arm-sme.mlir

```diff
@@ -1320,3 +1320,35 @@ func.func @vector_extract_element_f64(%row: index, %col: index) -> f64 {
   %el = vector.extract %tile[%row, %col] : f64 from vector<[2]x[2]xf64>
   return %el : f64
 }
+
+// -----
+
+// CHECK-LABEL: @dynamic_vector_extract_mask_to_psel(
+// CHECK-SAME:                                       %[[A:[a-z0-9]+]]:  index,
+// CHECK-SAME:                                       %[[B:[a-z0-9]+]]: index,
+// CHECK-SAME:                                       %[[INDEX:[a-z0-9]+]]: index)
```

Comment:
This is required as `.*: index,` matches all of `%arg0: index, %arg1: index,` (A = `arg0: index, %arg1`).
See: https://regex101.com/r/W0bhuJ/2 vs https://regex101.com/r/TKIcKM/1

Reply:
This would work:
```
// CHECK-SAME:  %[[A:.*]]:  index, %[[B:.*]]: index, %[[INDEX:.*]]: index)
```
Or just update the other regex'es in the file, there's not that many 🤔 Just a nit.